### ISSUE TITLE
LESS: list breaks into new lines

### ISSUE THREAD

**Issue by tilwinjoy** [beautify-web/js-beautify#953]
If I type the following:

``` less
@colors: red, green, blue;
```

It becomes:

``` less
@colors: red,
green,
blue;
```

It should stay as it is, or at least there should be an option to prevent such breaking.

---

I'm using [brackets-beautify v2.1.0](https://github.com/brackets-beautify/brackets-beautify/releases) which in turn uses js-beautify 1.6.3

**Comment by bitwiseman** [comment]
Is this part of a larger LESS file? I want to make sure this is tested correctly.

**Comment by bitwiseman** [comment]
@tilwinjoy - Ping?

**Comment by tilwinjoy** [comment]
Hi, sorry about the late reply. Just tested it with a new `.less` file (no other content) and it still breaks this way...

**Comment by SamPotts** [comment]
Same for me for font stacks. I had to escape the values which is not ideal but works for now:

```
@variable: ~"'value', value, value";
```

**Comment by anyclub** [comment]
me too

**Comment by MacKLess** [comment]
This appears to have been resolved as of v1.8.0-rc13. @tilwinjoy, @sampotts, and @anyclub please double check and let me know if you disagree.
